### ISSUE TITLE
Crush at Room 10D playing as Claire in Jill custome

### ISSUE THREAD

**Issue by HaithamGaafer** [IntelOrca/biorand#274]
### Version

2.3.0 (5bbf478)

### Seed

R720-FTC8-ZJNNA0QZLYLM0LRVGLF9ZZZZZZZZZZZZ0CZ

### Scenario

RE 2 (Claire B)

### What happened?

When I was entering from the library room, the game crashed at this room.

**Comment by IntelOrca** [comment]
I am able to reproduce this. Can you please provide more information, for example what you were doing at the time or steps on how to get it to crash if you jump to the room from the start of the game.

A saved game can also help.

**Comment by HaithamGaafer** [comment]
I am sorry I don't think I have a save data as I was able to bypass this
room by using the debug menu and using the generated log to see which room
it's connected to...

The game just kept on crashing whether I teleported there using the debug
menu or entering through either doors connected to this room.
All what I remember is that I was playing as Jill in claire b scenario.
Sorry if I can't provide more information.

By the way, Great mod!! I am really enjoying it! Thank you for your time
and effort you poured into it!
